subsys/usb/class/cdc_acm.c

```diff
@@ -422,9 +420,11 @@ static int cdc_acm_fifo_fill(struct device *dev,
 	len = len > sizeof(u32_t) ? sizeof(u32_t) : len;
```

Comment:
Not relevant to the change made below. The Quark limitation can be removed here as it is taken care in the usb_dc_dw.c driver now. @finikorg Could you please confirm?

Reply:
@sundar-subramaniyan I thought it should but we have one issue I am currently investigating #5678. Let's keep this for a while.